hexa/pipelines/urls.py

```diff
@@ -15,4 +15,10 @@
     ),
     path("credentials/", views.credentials, name="credentials"),
     path("status", views.pipelines_status, name="status"),
+    path(
```

Comment:
When I look at the rest of the code, I'm starting to think that this could be a query string parameter as well.